.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+## Title
+
+<!-- Update the following example details to be relevant -->
+
+PHP version: `8.1`  
```

Comment:
```suggestion
PHP version: `8.1`
```

Reply:
I put that there for valid markdown notation to indicate new line please.
So it looks like this
<img width="847" alt="Screenshot 2023-09-14 at 11 36 19 AM" src="https://github.com/silverstripeltd/bespoke-standards/assets/11827838/35398a47-d255-4529-a77d-0d310f6e1610">

not
<img width="926" alt="Screenshot 2023-09-14 at 11 36 37 AM" src="https://github.com/silverstripeltd/bespoke-standards/assets/11827838/cfabf7ef-4ae6-48f2-adb9-9fd74720be91">

Reply:
ah, ok...yeah, I remember you mentioning that - not sure I will ever get used to it tho